.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+php:
+  - 7.1
+  - 7.2
+
+matrix:
+  include:
+    - php: 7.1
+      env: dependencies="--prefer-lowest"
+    - php: 7.1
+      env: CHECK_CODESTYLE=1
+      before_install: phpenv config-rm xdebug.ini
+      script: composer check
+      after_success: ~
+
+install:
+  - travis_retry composer self-update && composer --version
+  - travis_retry composer update --no-interaction $DEPENDENCIES
+
+script:
+  - vendor/bin/phpunit --coverage-clover coverage-clover.xml
```

Comment:
osobne mam radsi, kdyz je tohle nejak zabalene pod `composer test`  napr

Reply:
Jo, jenze tohle je specificky nastaveni pro travis (coverage se jinak negeneruje).

A zabalovat vendor/bin/phpunit do "composer test" mi neprijde moc ucelny :)

Reply:
no ja v ramci toho testu beru i code style check a dalsi veci.. zkratka celou "kompilaci"
- navic mi prijde mozna zbytecne nemit ten code-coverage primo v configu pro phpunit

Reply:
No to prave typicky nemusis chtit, protoze coverage se vetsinou generuje jen na CI, protoze to jinak zdrzuje (a musis mit lokalne v CLI xdebug, coz nechces).

To se da resit pres phpunit.xml.dist a lokalni phpunit.xml , ale prijde mi lepsi mit to takhle obracene - by default vypnuty, na CI explicitne zapnuty.

Reply:
no ja zase prave ve stormu kdyz poustim testy, tu coverage chci, aby mi obarvil kod, ktery prosel

Reply:
pak ti ale nic nebrani udelat si `composer cli`, kde budes mit tyhle casy poresene a mas nad tim kontrolu

Reply:
Jasne, ale mne jde o minimalni default, aby kdyz pustim vendor/bin/phpunit, tak to spustilo testy, a nedelalo zadny dalsi kejkle, nepsalo chyby ze chybi xdebug atd.

Pokud potrebuje nekdo lokalne neco specifickyho, necht si udela lokalni phpunit.xml nebo to spusti se specifickym CLI prikazem. Ale tezko najit kompromis, kdyz to kazdy pouziva jinak, nez "minimalni varianta", ktera tu je.

Reply:
jo to asi klidne, na druhou stranu, kdyz tady zname presne dany use-case, proc ho nedefinovat konkretne do `composer cli` prikazu

Reply:
Toto je ale specificke chovani prave a jenom tohoto buildu pro Travis, nikoliv obecne pro CLI ani pro nic jineho - tudiz .travis.yml mi prijde jako to spravne misto na to, kde to mit definovane :).